src/main/java/net/smartcosmos/extension/tenant/rest/resource/tenant/CreateTenantResource.java

```diff
@@ -32,7 +29,6 @@
 @SmartCosmosRdao
 @Slf4j
 @ConditionalOnProperty(prefix = ENDPOINT_ENABLEMENT_TENANTS, name = ENDPOINT_ENABLEMENT_PROPERTY_ENABLED, matchIfMissing = true)
-@PreAuthorize("permitAll()")
```

Comment:
I'm really a fan of this explicit `@PreAuthorize`, because we indicate that we didn't forget to set the required authorization, but rather want to allow all users to access this endpoint.